lib/appium_capybara/driver/appium/driver.rb

```diff
@@ -78,6 +78,11 @@ def scroll_down
       browser.execute_script('mobile: scroll', direction: 'down')
     end
 
+    # new
+    def swipe(start_x, start_y, end_x=0, end_y=0, duration=200)
```

Comment:
Could you make arguments as keywrod arguments like `swipe(start_x:, start_y:, end_x: 0, end_y: 0, duration: 200)` or define here as hash lilke https://github.com/appium/ruby_lib_core/blob/fd2cbde5466a144ce8dccfc6cf330b969bcc8fd3/lib/appium_lib_core/common/touch_action/touch_actions.rb#L158-L172 ?

Reply:
sure; on it

Reply:
@KazuCocoa updated and tested locally; it works good with the arguments as keywords. added description in this PR